k2/csrc/fsa_algo_test.cc

```diff
@@ -19,10 +19,13 @@
 namespace k2 {
 
 TEST(FsaAlgo, Connect) {
-  std::vector<Arc> arcs = {
-      {0, 1, 1}, {0, 2, 2}, {1, 3, 3}, {1, 6, 6},
-      {2, 4, 2}, {2, 6, 3}, {5, 0, 1},
-  };
+  std::vector<Arc> arcs = {{0, 1, 1},
```

Comment:
`StringToFsa` can help you build the fsa.

Reply:
Actually I dont think it's a good idea to use StringToFsa in testing, as it only save the line for defining `arc_indexes` but we cannot know what `arc_indexes` is by reading code.
However, I think `StringToFsa` will be used in the future for reading Fsa from files.